ui/rla_ui.js

```diff
@@ -165,9 +169,12 @@ function maybeGetAuditType(andThen) {
    if ( ! uiState['got_audit_type']) {
       var auditType = conductorState['audit_type_name'];
       if (auditType === null) {
-         chooseAuditType();
+         uploadBallotManifestJson('ballot_comparison'); // call upload JSON function 
+       
       } else {
-         displayAuditType();
+         displayJSONFile();
```

Comment:
I think these each need their own `if ( ! uiState['got_audit_type']) {`-style if-statement. The idea is that if we reload the page it'll bring us to the right spot (and we won't need to restart our work).